app/bundles/FormBundle/Controller/ResultController.php

```diff
@@ -392,6 +394,218 @@ public function deleteAction(Request $request)
         );
     }
 
+    public function markSpamAction(Request $request, Configurator $configurator): \Symfony\Component\HttpFoundation\RedirectResponse|Response
+    {
+        $formId   = $request->get('formId', 0);
+        $objectId = $request->get('objectId', 0);
+        $session  = $request->getSession();
+        $page     = $session->get('mautic.formresult.'.$formId.'.page', 1);
+        $flashes  = [];
+
+        if (Request::METHOD_POST === $request->getMethod()) {
+            $model = $this->getModel('form.submission');
+            \assert($model instanceof SubmissionModel);
```

Comment:
Please use dependency injection
```suggestion
    public function markSpamAction(Request $request, Configurator $configurator, SubmissionModel $model): \Symfony\Component\HttpFoundation\RedirectResponse|Response
    {
        $formId   = $request->get('formId', 0);
        $objectId = $request->get('objectId', 0);
        $page     = $request->getSession()->get('mautic.formresult.'.$formId.'.page', 1);
        $flashes  = [];

        if (Request::METHOD_POST === $request->getMethod()) {
```